makefile

```diff
@@ -45,15 +45,18 @@ libdex: cbits/libdex.so
 
 # --- running tets ---
 
-example-names = type-tests eval-tests shadow-tests annot-tests linear-tests ad-tests \
-                monad-tests include-test mandelbrot pi sierpinsky \
-                regression brownian_motion chol particle-swarm-optimizer \
+example-names = uexpr-tests type-tests eval-tests shadow-tests monad-tests \
+                easy-ad-tests mandelbrot pi sierpinsky \
+                regression brownian_motion particle-swarm-optimizer \
                 ode-integrator
+
+# example-names = linear-tests ad-tests include-test chol
 quine-test-targets = $(example-names:%=run-%)
 
 doc-names = $(example-names:%=doc/%.html)
 
-tests: test-prep quine-tests repl-test
+# tests: test-prep uexpr-tests quine-tests repl-test
```

Comment:
I guess this and the duplicate `example-names` are no longer necessary?

Reply:
Yes, though it's good to keep track of tests that we've temporarily disabled. Added TODOs.